sdks/python/setup.cfg

```diff
@@ -25,4 +25,5 @@ verbosity=2
 # fast_coders module which is not available when running unit tests:
 # fast_coders_test and typecoders_test.
 exclude=fast_coders_test|typecoders_test
-
+# Include end-to-end tests.
+include=wordcount_it
```

Comment:
Could you try adding `_test` and checking that? I think it is fine to add a `_test` to the name if that works. Otherwise all integration tests would need to be explicitly included here.